packages/terra-form-input/src/terra-dev-site/test/form-input/InputField.test.jsx

```diff
@@ -11,6 +11,11 @@ class InputFieldExample extends React.Component {
     };
 
     this.toggleInvalid = this.toggleInvalid.bind(this);
+    this.refCallback = this.refCallback.bind(this);
+  }
+
+  refCallback(input) {
+    this.input = input;
```

Comment:
Should we do something with this callback to demonstrate a use case?

Reply:
I'll do the autofocus like I did with the other tests